src/dbnode/client/config.go

```diff
@@ -310,6 +312,19 @@ func (c Configuration) NewAdminClient(
 
 	if c.Proto != nil && c.Proto.Enabled {
 		v = v.SetEncodingProto(encodingOpts)
+		schemaRegistry := namespace.NewSchemaRegistry(true, nil)
+		// Load schema from config if it is available.
+		// If admin client is initialized by dbnode, the schema registry can be overridden
+		// by admin custom options.
+		for nsID, protoConfig := range c.Proto.SchemaRegistry {
+			dummyDeployID := "fromconfig"
```

Comment:
super nit: Can we make this a constant in some package since we use it on the server side and client side? Would be nice to have those match

Reply:
I have a follow up pr that require client side config have schema deploy ID set. (to the one that is the real deploy ID)